var/spack/repos/builtin/packages/py-ipywidgets/package.py

```diff
@@ -37,4 +38,6 @@ class PyIpywidgets(PythonPackage):
     depends_on('py-widgetsnbextension@3.4.0:3.4', type=('build', 'run'),
                when='@7.4.2')
     depends_on('py-widgetsnbextension@3.5.0:3.5', type=('build', 'run'),
-               when='@7.5.1:')
+               when='@7.5.1:7.6.5')
+    depends_on('py-widgetsnbextension@3.6.0', type=('build', 'run'),
+               when='@7.7.0')
```

Comment:
```suggestion
               when='@7.7:')
```
I think it's safe to assume this requirement will also exist in newer versions